addons/sale_timesheet/migrations/11.0.1.0/post-migration.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2018 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def migrate_sale_line_tasks(env):
+    openupgrade.logged_query(
+        env.cr, """
+            UPDATE sale_order_line sol
+            SET task_id = po.task_id
+            FROM procurement_order po
+            WHERE sol.id = po.sale_line_id
+        """)
+
+
+def migrate_project_sale_line(env):
+    openupgrade.logged_query(
+        env.cr, """
+            UPDATE project_project pj
+            SET sale_line_id = sol.id
+            FROM sale_order_line sol, product_product pd, sale_order so,
+              product_template pt
+            WHERE sol.product_id = pd.id
+            AND sol.order_id = so.id
+            AND so.analytic_account_id = pj.id
+            AND pt.id = pd.product_tmpl_id
+            AND so.state in ('sale', 'done')
+            AND pt.service_tracking = 'task_new_project'
```

Comment:
You can't put this condition. This is a new value in v11 and no product will have this. Remember how a new project was generated in v10. It was leaving the project field in blank and selecting the option to create new task

Reply:
Ok, I understand.